ci-operator/config/openshift/csi-driver-shared-resource/openshift-csi-driver-shared-resource-master.yaml

```diff
@@ -15,20 +15,17 @@ binary_build_commands: make build
 build_root:
   from_repository: true
 images:
-- context_dir: images
-  dockerfile_path: Dockerfile
+- dockerfile_path: images/Dockerfile
   from: base
   inputs:
     ocp_builder_rhel-8-golang-1.17-openshift-4.11:
       as:
       - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.17-openshift-4.11
   to: csi-driver-shared-resource
-- context_dir: images
-  dockerfile_path: Dockerfile.mustgather
+- dockerfile_path: images/Dockerfile.mustgather
   from: must_gather
   to: csi-driver-shared-resource-mustgather
-- context_dir: images
-  dockerfile_path: Dockerfile.webhook
+- dockerfile_path: images/Dockerfile.webhook
```

Comment:
Yeah I'm curious if we really need 
```
  inputs:
    ocp_builder_rhel-8-golang-1.17-openshift-4.11:
      as:
      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.17-openshift-4.11
```

like we have for the driver image 

let's go without it like you have here, see what happens